consul/base.py

```diff
@@ -1018,29 +1034,37 @@ def register(
                     params=params,
                     data=json.dumps(payload))
 
-            def deregister(self, check_id):
+            def deregister(self, check_id, token=None):
                 """
                 Remove a check from the local agent.
                 """
+                params = []
```

Comment:
params is never used here

Reply:
It is, see 3 lines after.

Reply:
Got it now, not used in the http method.
Fixed now.